cmd/erigon-cl/core/transition/beacon_state_utils.go

```diff
@@ -142,6 +143,15 @@ func ComputeSigningRootEpoch(epoch uint64, domain []byte) ([32]byte, error) {
 	return utils.Keccak256(hash[:], domain), nil
 }
 
+func ComputeSigningRoot(obj ssz_utils.ObjectSSZ, domain []byte) ([32]byte, error) {
```

Comment:
I think we already have this in fork package

Reply:
fork.ComputeSigningRoot

Reply:
thanks, done!